internal/command/certificates/root.go

```diff
@@ -161,7 +161,9 @@ func runCertificatesShow(ctx context.Context) error {
 	}
 
 	if cert.ClientStatus == "Ready" {
-		fmt.Fprintf(io.Out, "The certificate for %s has been issued.\n\n", colorize.Bold(hostname))
+		if !flag.GetBool(ctx, "json") {
+			fmt.Fprintf(io.Out, "The certificate for %s has been issued.\n\n", colorize.Bold(hostname))
+		}
```

Comment:
this is not the only place where "printCertificate" is called, and the messages about certificate issued/not issued are printed on all cases. Simpler to move them to the printCertificates function so it works for json in all cases.

Reply:
Thanks for your reply. I'm not sure how to go about this though. I'd need to make printCertificates know if it should say "has been issues" or "has not been issued". Should I simply make it take a boolean as third argument and adjust the calls of printCertificates?

Reply:
no need to pass a third argument, `cert` is already passed to printCertificates and for _issued/not issued_ it checks `cert.ClientStatus == "Ready"` field

Reply:
You are right, thank you. It should be better now.